src/main/java/duke/task/Deadlines.java

```diff
@@ -0,0 +1,30 @@
+package duke.task;
+
+import duke.util.DateAndTimeFormatter;
+
+import java.time.LocalDate;
+
+public class Deadlines extends Task {
```

Comment:
The classes extending from Task should be singular instead of plural since they represent a single entity.